.github/workflows/test-status-check.yml

```diff
@@ -15,7 +15,7 @@ on:
 jobs:
   main_tests:
     name: Run ${{ vars.PROJECT_NAME }} Tests
-    uses: KinsonDigital/Infrastructure/.github/workflows/run-csharp-tests.yml@v13.0.1
+    uses: KinsonDigital/Infrastructure/.github/workflows/run-csharp-tests.yml@v13.1.0
```

Comment:
Need to add the `checkout-ref` and `checkout-repository` workflow inputs.
Make sure to use Velaptor as a guide.  We implemented it wrong with the CASL project.